code/modules/admin/verbs/randomverbs.dm

```diff
@@ -1183,6 +1228,19 @@ Traitors and the like can also be revived with the previous role mostly intact.
 
 	punish_log(target, punishment)
 
+/proc/firing_squad(mob/living/carbon/C, turf/open/OT, body_zone, wound_bonus, damage)
+	if(!C.get_bodypart(body_zone))
+		return
+	playsound(C, 'sound/weapons/gun/revolver/shot.ogg', 100)
+	var/obj/projectile/bullet/smite/divine_wrath = new(OT)

```

Comment:
god imagine if someone manages to trick an admin into using this specific smite on them while they know sleeping carp, that'd be badass as hell

(keep it in)